package-lock.json

```diff
@@ -14375,514 +14379,440 @@
       }
     },
     "node_modules/esbuild": {
-      "version": "0.21.2",
-      "resolved": "https://registry.npmjs.org/esbuild/-/esbuild-0.21.2.tgz",
-      "integrity": "sha512-LmHPAa5h4tSxz+g/D8IHY6wCjtIiFx8I7/Q0Aq+NmvtoYvyMnJU0KQJcqB6QH30X9x/W4CemgUtPgQDZFca5SA==",
+      "version": "0.25.1",
```

Comment:
The version here went up, but the `package.json` claims a downgrade to `0.19.11`?